inductiva/localization/localization.py

```diff
@@ -141,7 +141,7 @@ def __getitem__(self, key: str) -> str:
             key (str): Key for the translation to get.
         """
         translations = self._translations.get(self._lang)
-        if key not in translations:
+        if translations is None or key not in translations:
```

Comment:
strange, I wonder what the value of `self._lang` is.

Reply:
At this point, if `locale` is not set, `self._lang` should be `_INDUCTIVA_DEFAULT_LANG` 🤔

Reply:
Cannot address the lang part now, but the highlight is that the `translation` was None. That maybe pointing to a deeper bug where the language is set correctly, but it may not appropriately refer to the language we have. For example, it could be `pt-br`.

@Pbarbosa92 can you try on your windows computer what is the language here? Just put a print for the self._lang and call a terminal command of the terminate a resource.

Reply:
Changing the default language on the Mac keeps working and it prints `self._lang` as pt, so it needs to be something more inherent of Windows.